bip-emilengler-redefinition-of-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Redefinition of the term address
```

Comment:
Maybe title should be "Replacing the term address" if the goal isn't to give the word "address" a new meaning, but to introduce new terms that should be used instead

Reply:
ACK